.github/ISSUE_TEMPLATE/010-specification-review.md

```diff
@@ -10,41 +10,46 @@ assignees: ''
 
 Hello TAG!
 
-I'm requesting a TAG review of:
+I'm requesting a TAG review of [feature name].
 
-  - Name: [feature or document name]
+[One paragraph summary of idea, ideally copy-pasted from Explainer introduction]
+
+  - Explainer¹ (minimally containing user needs and example code): [url]
   - Specification URL: [spec url]
-  - Explainer (containing user needs and example code)¹: [url]
-  - GitHub issues (if you prefer feedback filed there): [url]
   - Tests: [wpt folder(s), if available]
-  - Primary contacts (and their relationship to the specification): [github usernames and affiliation]
-  - Organization/project driving the specification: [organization or project name] 
+  - Security and Privacy self-review²: [url]
+  - GitHub repo (if you prefer feedback filed there): [url]
+  - Primary contacts (and their relationship to the specification): 
+      - [name] ([github username]), [organization/s] (repeat as necessary)
+  - Organization(s)/project(s) driving the specification: [organization and/or project name] 
+  - Existing major pieces of multi-stakeholder review or discussion of this specification: 
```

Comment:
Concerned this may be overly draconian? Maybe drop "major" or change to "evidence of multi-stakeholder review, discussiomn or support of this specification:" ?

Reply:
Maybe replace "major" with "main" or "primary"?  I think the idea is that we want to see how much review there was, so we want to see the largest pieces, however large or small they are.

Reply:
Might go with "key".